plugins/sample/plugin_sample.c

```diff
@@ -23,10 +23,10 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
+#include <stdio.h>
```

Comment:
Which compiler warning flag caused this one?

Reply:
Not a warning per se on this one, but making the order the same as it is in the dummy plug-in.